doc/sklearn.rst

```diff
@@ -40,10 +40,15 @@ Patches can also be enabled programmatically::
     import daal4py.sklearn
     daal4py.sklearn.patch_sklearn()
 
+If some modules from scikit-learn were imported before patch_sklearn(),
+then you need to re-import these modules after patch_sklearn().
```

Comment:
```suggestion
you have to reimport these modules after ``patch_sklearn()``.
```

Reply:
since you repeat the name of the function twice in one sentence, you might want to rephrase it, e.g. something like this:

```suggestion
you have to reimport these modules again after patching.
```

or

```suggestion
you have to reimport these modules again after enabling the patches.
```